html/semantics/forms/the-input-element/size.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<link rel="author" title="Domenic Denicola" href="mailto:d@domenic.me">
+<link rel="help" href="https://html.spec.whatwg.org/multipage/#attr-input-size">
+<link rel="help" href="https://html.spec.whatwg.org/multipage/#limited-to-only-non-negative-numbers-greater-than-zero">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+
+<input>
+
+<script>
+"use strict";
+
+test(() => {
+    const input = document.querySelector("input");
```

Comment:
Yeah; someone else will have to carry these tests across the finish line in that case.